sdk/keyvault/azkeys/models.go

```diff
@@ -194,14 +194,38 @@ func jsonWebKeyFromGenerated(i *generated.JSONWebKey) *JSONWebKey {
 		return &JSONWebKey{}
 	}
 
+	ops := make([]*Operation, len(i.KeyOps))
+	for j, op := range i.KeyOps {
+		if op == nil {
+			ops[j] = nil
+		} else {
+			switch *op {
+			case string(OperationDecrypt):
+				ops[j] = to.Ptr(OperationDecrypt)
```

Comment:
Seels like `op` is already a pointer-to-type, can't we just use its value?

Reply:
Yes, there's no need to confine the values to the set of defined constants.